### ISSUE TITLE
Rule Tester not able to serialize numpy data types

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/cdisc-rules-engine#489]
[CG0213.zip](https://github.com/cdisc-org/cdisc-rules-engine/files/12198535/CG0213.zip)

See attached test data. The issue is caused by the following:

```yaml
Operations:
  - domain: TV
    id: $visitnums_in_tv
    name: VISITNUM
    operator: distinct
  Output Variables:
    - SVUPDES
    - VISITNUM
    - $visitnums_in_tv
```

$visitnums_in_tv contains numpy int64 types that cannot be serialized to json  here: https://github.com/cdisc-org/cdisc-rules-engine/blob/2628d82b87acafcad3d04178c9aa17b32ebfe80d/TestRule/__init__.py#L84

This causes an exception.

Numpy types should be encoded and serialized so that an exception is not raised.

**Comment by SFJohnson24** [comment]
this is resolved with https://github.com/cdisc-org/cdisc-rules-engine/pull/1361
